### ISSUE TITLE
Add version of Point#d.centroid that does not return a Maybe?

### ISSUE THREAD

**Issue by ianmackenzie** [ianmackenzie/elm-geometry#87]
Following on from #64 - would likely have signature

```
Point#d -> List Point#d -> Point#d
```

(use two arguments to emulate "non empty list") but what to call it? Possibilities include:

  - `centroidFrom`
  - `nonEmptyCentroid`
  - `definiteCentroid`
  - `consCentroid`

Could also rename the existing function to `maybeCentroid` and then call this one `centroid`, but then would `Polyline#d.centroid` etc. also get renamed to `maybeCentroid`?

**Comment by ianmackenzie** [comment]
If by chance you're here from [this Discourse thread](https://discourse.elm-lang.org/t/proposal-pattern-for-dealing-with-empty-lists/3792): if people agree that that is a good pattern to follow, then `Point#d.centroid` itself would be changed to have the signature
```elm
Point#d -> List Point#d -> Point#d
```
and no second function would need to be added (since `List.ifNotEmpty` would then be used to handle the 'possibly empty list' case).

**Comment by ianmackenzie** [comment]
Implemented in fcdf0c2030164c19442d45cbc196f857bb57d133
